pkg/preprocessing/chat_completions_template/chat_template_test.go

```diff
@@ -0,0 +1,619 @@
+//go:build exclude
+
+/*
+Copyright 2025 The llm-d Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package chat_completions_template
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Global singleton wrapper to prevent multiple Python interpreter initializations
+var (
```

Comment:
Same comment as in `cgo_functions.go` - I think these variables and their use can be emitted since the C code ensures single initialization per process.